source/aggregation-pipeline-builder.txt

```diff
@@ -112,35 +112,31 @@ your chosen view mode:
       .. procedure:: 
          :style: normal
 
-         .. step:: In the aggregation pipeline pane, ensure the 
+         .. step:: Select the :guilabel:`Stage` view.
+            
+            In the aggregation pipeline pane, ensure the 
             :guilabel:`{} Stages` toggle switch is selected.
 
             .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
                :alt: Toggle on stage view mode 
 
-         .. step:: At the bottom of the aggregation pipeline pane, select the 
```

Comment:
[praise] I really like the way you've broken the steps out into action & more information below! Nice work.